x-pack/legacy/plugins/siem/public/components/source_destination/country_flag.tsx

```diff
@@ -47,3 +47,32 @@ export const CountryFlag = memo<{
 });
 
 CountryFlag.displayName = 'CountryFlag';
+
+/** Renders an emjoi flag with country name for the specified country code */
+export const CountryFlagAndName = memo<{
+  countryCode: string;
+  displayCountryNameOnHover?: boolean;
+}>(({ countryCode, displayCountryNameOnHover = false }) => {
+  const [localesLoaded, setLocalesLoaded] = useState(false);
+  useEffect(() => {
+    if (isEmpty(countries.getNames('en'))) {
```

Comment:
Should we be using the `format:number:defaultLocale` Kibana advanced setting for all these `countries.getNames()` calls?


e.g. 
``` ts
console.log("US (Alpha-2) => " + countries.getName("US", "en")); // United States of America
console.log("US (Alpha-2) => " + countries.getName("US", "de")); // Vereinigte Staaten von Amerika
```

from the [i18n-iso-countries docs](https://github.com/michaelwittig/node-i18n-iso-countries#get-the-name-of-a-country-by-its-iso-3166-1-alpha-2-alpha-3-or-numeric-code).

Reply:
We then need to load all languages. I think we talked about this @cwurm but can't remember why we ended up with english only??

Reply:
Hmmm... If that's the case, we could use  [typescript's dynamic import](https://mariusschulz.com/blog/dynamic-import-expressions-in-typescript) to resolve that issue pretty easily.

As for not previously using it -- perhaps it was because country names are often stored as their English counterpart in Elasticsearch directly? Doesn't look like we're using the localized country name in any DnD/AddToKql components, so that should be fine too?

Either way, just something to note 🙂